### PR TITLE
Add lawn mower entity state, icon and color

### PR DESCRIPTION
**Pull request by mikey0000**
## Proposed change

Adding lawn mower entity to frontend

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

Link to core integration pull request: https://github.com/home-assistant/core/pull/93623
Link to brands pull request: https://github.com/home-assistant/brands/pull/4561
Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/28546
Link to developer documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1875

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/common/entity/state_color.ts
Comment (mikey0000) [reply src/common/entity/state_color.ts:25]: I'll take a look and set them :)

Comment (mikey0000) [reply src/common/entity/state_color.ts:25]: I've picked green for mowing and teal for docking. Let me know if green is not a good choice for "in-work" operations from an overall theme and design perspective. I've worked on design systems before so if this doesn't work, just let me know. Also green and red I know aren't great for colour blind individuals, though my experience with the shades HA has is of course quite limited. So open to changing green to something else to accommodate for accessibility here.

Comment (piitaya) [reply src/common/entity/state_color.ts:25]: We can keep teal for active state (mowing and docking), red for error and off state (docked and paused) will be grey out. WDYT?

**Comment by piitaya** [comment]
Also, some UI elements will be needed (entity row, more info). Do you want me to take this task or do you prefer to do it? It can be in another PR after this one.

File: src/common/entity/state_color.ts
Comment (mikey0000) [reply src/common/entity/state_color.ts:25]: docked I would expect to be the default blue colour, off = grey I agree with.

Comment (mikey0000) [reply src/common/entity/state_color.ts:25]: Oh and yes I'll change green to teal, I'm not too fussed here right now, can always review it later.

Comment (mikey0000) [reply src/common/entity/state_color.ts:25]: Yes docking equiv of return to home.

**Comment by mikey0000** [comment]
> 



> Also, some UI elements will be needed (entity row, more info). Do you want me to take this task or do you prefer to do it? It can be in another PR after this one.

I would love it if you could take this on. I have no experience with HA's frontend framework.

**Review by piitaya — APPROVED** [review]
Thank you 🙂 It can be merged once back end merged

**Pull request merged**
